src/json-patch-queue-synchronous.js

```diff
@@ -59,12 +66,12 @@ JSONPatchQueueSynchronous.prototype.receive = function(obj, versionedJsonPatch,
 	// consecutive new version
 		while( consecutivePatch ){// process consecutive patch(-es)
 			this.version++;
-			apply(obj, consecutivePatch);
+			apply(this.obj, consecutivePatch);
```

Comment:
Don't we need `this.obj = apply(...` here as well?

Reply:
Good point.